edb/lib/sys.edgeql

```diff
@@ -443,3 +443,8 @@ sys::approximate_count(
     USING SQL FUNCTION 'edgedb.approximate_count';
     set impl_is_strict := false;
 };
+
+
+CREATE PERMISSION sys::data_modification;
+CREATE PERMISSION sys::ddl;
+CREATE PERMISSION sys::persistent_config;
```

Comment:
Can we move these into a sub-namespace? `sys::perm::*`?

Reply:
Yeah why not

Reply:
https://github.com/geldata/rfcs/pull/115